wifiphisher/pywifiphisher.py

```diff
@@ -549,6 +549,7 @@ def __init__(self):
         self.template_manager = phishingpage.TemplateManager()
         self.access_point = accesspoint.AccessPoint()
         self.fw = firewall.Fw()
+        self.em = extensions.ExtensionManager()
```

Comment:
@sophron you should do `em` exit before calling `network_manager.on_exit`. `network_manager.on_exit` will set the card back to managed mode, thus will down the card. This will cause exception for scapy.